lib/client.c

```diff
@@ -404,6 +410,17 @@ _snowflake_check_connection_parameters(SF_CONNECT *sf) {
         return SF_STATUS_ERROR_GENERAL;
     }
 
+    if (SF_BOOLEAN_FALSE == validate_application(sf->application)) {
+        // Invalid connection
```

Comment:
nit: Should say its an invalid application name that was passed in

Reply:
@sfc-gh-kwagner done.